src/test/java/seedu/address/logic/commands/patient/ClearPatientCommandTest.java

```diff
@@ -23,13 +26,21 @@ public void execute_emptyPatientRecords_success() {
 
     @Test
     public void execute_nonEmptyPatientRecords_success() {
-        Model model = new ModelManager(getTypicalAppointmentSchedule(), getTypicalPatientRecords(),
+        Model model = new ModelManager(getEmptyAppointmentSchedule(), getTypicalPatientRecords(),
                 new UserPrefs());
-        Model expectedModel = new ModelManager(getTypicalAppointmentSchedule(), getTypicalPatientRecords(),
+        Model expectedModel = new ModelManager(getEmptyAppointmentSchedule(), getTypicalPatientRecords(),
                 new UserPrefs());
         expectedModel.setPatientRecords(new AddressBook<>());
 
         assertCommandSuccess(new ClearPatientCommand(), model, ClearPatientCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void execute_nonEmptyPatientRecords_failure() {
```

Comment:
same here, method name is wrong, should be `execute_nonEmptyAppointmentScheduleNonEmptyPatientRecords_failure` or something along those lines